src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderResponseContainer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+using SwedbankPay.Sdk.Exceptions;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrderResponseContainer
+    { 
+        public PaymentOrderResponseContainer(PaymentOrderResponse paymentOrder)
+        {
+            PaymentOrder = paymentOrder;
+        }
+
+
+        public OperationList Operations { get; set; } = new OperationList();
```

Comment:
This operation list is never filled. Either way I'd recommend removing the setter, especially as we're initializing the list here.